src/software/simulated_tests/friendly_goal_scored.py

```diff
@@ -0,0 +1,51 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class FriendlyGoalScored(Validation):
```

Comment:
nit: can we update the name of this class + file? At first I thought it was saying that a goal was scored on the _friendly goal_. The naming of the validation functions are much more intuitive imo.

Reply:
suggestion: `FriendlyTeamHasScoredGoal`

Reply:
Should I also change the name of the file to friendly_team_has_scored_goal.py then (and wherever the file is referenced)?

Reply:
nvm this was part of the initial comment

Reply:
done